IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/SSNewsCollectionViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SSNewsCollectionViewCell.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+import SDWebImage
+
+final class SSNewsCollectionViewCell: UICollectionViewCell {
+
+    static let identifier = "SSNewsCollectionViewCell"
+    static let Height = 80.0
+    
+    @IBOutlet weak var imgDetail: UIImageView!
+    @IBOutlet weak var lblDate: UILabel!
+    @IBOutlet weak var lblDescription: UILabel!
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblAuthor: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.contentView.frame = CGRect(x: self.contentView.frame.origin.x, y: self.contentView.frame.origin.y, width:UIScreen.main.bounds.width , height: CGFloat(SSNewsCollectionViewCell.Height))
+    }
+
+    
+    func setup(viewModel: SSNewsCollectionCellVM)
+    {
+        imgDetail.sd_setImage(with: URL(string: viewModel.imageUrl), placeholderImage: UIImage(named: "placeholder.png"))
+        lblDate.text = "\(viewModel.date)"
+        lblDescription.text = viewModel.description
+        lblTitle.text = viewModel.title
+        lblAuthor.text = viewModel.author
+    }
+    
+    override func layoutSubviews()
+    {
+        imgDetail.layer.borderWidth = 1
+        imgDetail.layer.borderColor = UIColor.clear.cgColor
```

Comment:
Is this a transparent border with width 1? Is it OK? Sounds weird 😄 

(If you want a separator border to differentiate the image from the background, I think it should have a color, not transparent, but I'm not sure what was the intention here)